application/src/utils/validate/isEmailValid.test.ts

```diff
@@ -0,0 +1,20 @@
+import { isEmailValid } from './isEmailValid';
+
+describe('isEmailValid function', () => {
+  let email;
+  beforeEach(() => {
+    email = 0;
+  });
+  test('is email 11111sun$@mail.ru valid', () => {
+    email = isEmailValid('11111sun$@mail.ru');
+    expect(email).toBeFalsy();
+  });
+  test('is  email jbjhbbhbba@mail.ru valid', () => {
```

Comment:
please don't use expected values in name of test case

Reply:
fixed